tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "moduleResolution": "node",
 
     // Enhance Strictness
+    "strict": true,
```

Comment:
I think you can remove the other "strict" compiler options (like `noImplicitAny`) since `stricts` activates them all anyway.

Reply:
Nice! I didn’t realize that. Do you know where the docs are for that (just so I can read up on it)?

Reply:
@rwjblue https://www.typescriptlang.org/docs/handbook/compiler-options.html